modules/apps/portal-security/portal-security-ldap-api/src/main/java/com/liferay/portal/security/ldap/configuration/BaseConfigurationProvider.java

```diff
@@ -46,11 +44,6 @@ protected String getMetatypeId() {
 		return _factoryPid;
 	}
 
-	protected abstract void setConfigurationAdmin(
-		ConfigurationAdmin configurationAdmin);
-
-	protected ConfigurationAdmin configurationAdmin;
```

Comment:
@W2jx Could you try to keep this field, add `@Reference` to it, and enable the portal-security-ldap-impl to inherit DS annotations? This is different with `ServletContext` because in that case, each sub-class references a unique servlet context; however, here all sub-classes are referencing the same `ConfigurationAdmin`.

Reply:
I will fix it.